### DISCUSSION TITLE
Support for FreeBSD

### DISCUSSION THREAD

**Discussion by almostunix** [ciderapp/Cider#984] [Rejected Features]
Hi, 

I use FreeBSD with Wayland/KDE Plasma for desktop. 

Was trying to follow the setup steps for Linux (I realize FreeBSD differs from Linux but thought it might still work... unfortunately it hangs during the "yarn install" step. 

It specifically fails at node_modules/electron with exit code 1

Appropriate versions of Node, yarn and python were installed prior to attempting to setup Cider. 

Any help is much appreciated, happy to provide screenshots or follow troubleshooting steps as needed.

**Comment by cryptofyre** [comment]
Our module from castlabs does not support FreeBSD at this time, id suggest contacting them about it. https://github.com/castlabs/electron-releases

**Reply by almostunix** [reply]
Thanks for the quick reply, that makes a lot of sense given where it got hung up.

**Reply by almostunix** [reply]
what version of electron is required for Cider to work correctly?

**Reply by cryptofyre** [reply]
Anything between 16.0.7-18.2.0 has been confirmed working. 18.2.0 being the one we use.

**Reply by almostunix** [reply]
Thanks. I have access to 16.0.9 for FreeBSD so I will see how I can incorporate that instead of the included one that is not supported yet on my OS. 

Thanks again.

**Comment by Amaru8** [comment]
Rejecting due to missing DRM support.
